ScoreSaber/UI/Elements/Profile/ProfileDetailView.cs

```diff
@@ -107,13 +180,47 @@ private HoverHint profileHoverHint {
                 return _profileHoverHint;
             }
         }
+
+        private bool isDownloadingMap { get; set; } = false;
         #endregion
 
         private PlayerService _playerService = null;
+        private BeatmapLevelsModel _beatmapLevelsModel = null;
+        private SoloFreePlayFlowCoordinator _soloFreePlayFlowCoordinator = null;
+        private PanelView _panelView = null;
+        private ScoreSaberLeaderboardViewController _scoreSaberLeaderboardViewController = null;
+
+        private IScoreSaberBeatmapDownloader _beatmapDownloader => Plugin.Container.TryResolve<IScoreSaberBeatmapDownloader>();
+
+        private Sprite _onlineSprite = null;
+        private Sprite onlineSprite {
+            get {
+                if (_onlineSprite == null) {
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
same here, we should fix the warning instead of just disabling it.

Reply:
fixed